coverage.py

```diff
@@ -89,7 +92,8 @@ def __init__(self, collection, uploader=None,
         # Since we are the metadata wrangler, any resources we find,
         # we mirror to S3.
         if not uploader:
-            uploader = S3Uploader()
+            uploader = S3Uploader.from_config(self._db)
```

Comment:
If the circ manager isn't using an S3 integration, it shouldn't ever be calling this method. How does an error raised in `from_config` get in the way of anything in that app?